lib/index.js

```diff
@@ -3,21 +3,39 @@
  * @typedef {import('unist').Position} UnistPosition
  * @typedef {import('vfile-message').VFileMessage} VFileMessage
  * @typedef {import('vscode-languageserver').Connection} Connection
- * @typedef {Partial<Pick<
- *   import('unified-engine').Options,
+ * @typedef {import('unified-engine').Options} EngineOptions
+ * @typedef {Pick<
+ *   EngineOptions,
  *   | 'ignoreName'
  *   | 'packageField'
  *   | 'pluginPrefix'
  *   | 'plugins'
- *   | 'processor'
  *   | 'rcName'
- * >>} Options
+ * >} EngineFields
+ *
+ * @typedef LanguageServerFields
+ * @property {string} processorName
+ *   The package ID of the expected processor (example: `'remark'`).
+ *   Will be loaded from the local workspace.
+ * @property {string} [processorSpecifier='default']
+ *   The specifier to get the processor on the resolved module.
+ *   For example, remark uses the specifier `remark` to expose its processor and
+ *   a default export can be requested by passing `'default'` (the default).
+ * @property {EngineOptions['processor']} [defaultProcessor]
```

Comment:
Why not use `import('unified').Processor`?

Anyway, if this is kept as-is, the `unified` dependency becomes unused and can be removed.

Reply:
They’re slightly different: https://github.com/unifiedjs/unified-engine/blob/19a498e9c42bf103b885b969e874c2790d4ead81/lib/index.js#L62.

Moved it to dev-deps